src/server/graphql/queries/helpers/makeUpcomingInvoice.js

```diff
@@ -4,7 +4,13 @@ import {fromEpochSeconds} from 'server/utils/epochTime';
 
 export default async function makeUpcomingInvoice(orgId, stripeId, stripeSubscriptionId) {
   if (!stripeId || !stripeSubscriptionId) return undefined;
-  const stripeInvoice = await stripe.invoices.retrieveUpcoming(stripeId, {subscription: stripeSubscriptionId});
+  let stripeInvoice;
+  try {
+    stripeInvoice = await stripe.invoices.retrieveUpcoming(stripeId, {subscription: stripeSubscriptionId});
+  } catch (e) {
+    // useful for debugging prod accounts in dev
```

Comment:
this is very true, i'll argue that it's safe to deploy for this 1 instance because this function is only used once where failing silently is actually desired (even in production), but it is a bad example for the rest of the codebase

Reply:
Alright, I'll take your word for it :)